### PR TITLE
Implemented getICCID() method

### PR DESCRIPTION
**Pull request by ubidefeo**

### CODE REVIEW

File: src/NBModem.cpp [hunk 1]
```diff
@@ -45,3 +45,15 @@ String NBModem::getIMEI()
 
   return imei;
 }
+
+String GSMModem::getICCID()
```
Comment (sandeepmistry) [inline src/NBModem.cpp]: Should be `String NBModem::getICCID()`

File: src/NBModem.cpp [hunk 2]
```diff
@@ -45,3 +45,15 @@ String NBModem::getIMEI()
...
+{
+  String iccid;
+
+  iccid.reserve(20);
+
+  MODEM.send("AT+CCID");
+  MODEM.waitForResponse(120, &iccid);
```
Comment (sandeepmistry) [inline src/NBModem.cpp]: Here's the result of the AT+CCID command for me:

```
AT+CCID

+CCID: 8901260882217768982

OK
```

We need to remove the `+CCID: ` ... something like
```
  if (iccid.startsWith("+CCID: ")) {
    iccid.remove(0, 7);
  } else {
    iccid = "";
  }
```

Comment (sandeepmistry) [reply src/NBModem.cpp]: @ubidefeo need to check if MKR GSM needs to same here.

Comment (ubidefeo) [reply src/NBModem.cpp]: yes, it does return the `+CCID: ` string, I think it's the norm.
I'm going to trim it out for both libs :)

Comment (ubidefeo) [reply src/NBModem.cpp]: one question, though...
why would you empty it out if it does not start with `+ICCID: `?
because it means it's invalid`?

Comment (sandeepmistry) [reply src/NBModem.cpp]: Yes, based AT command reference, it's expected to contain the `+ICCID: `

Comment (ubidefeo) [reply src/NBModem.cpp]: @sandeepmistry I'll add the else condition but sounds odd to me that an empty string be returned. Sounds more logical to return an error of some sort.
I do not know the customs, so I'll implement whatever you prefer :)

**Comment by sandeepmistry** [comment]
@SimonePDA the reference pages need to be updated for this.

**Pull request merged**